### DISCUSSION TITLE
Define Materials for different splice settings

### DISCUSSION THREAD

**Discussion by Samwise007** [tomvandeneede/p2pp#124] [Q&A] [answered]
Hey, I've been setting up my Palette 2S Pro with Prusaslicer. And so far it works great - thanks for that!
But now I have a print with 4 spools of PLA, and one of those needs a different splice setting - PLA in inputs 1, 2 and 3 need 0_0_0, but if one those gets spliced to the PLA in input 4 it should be 0_2_0.
I understood that I can define a material-override (;P2PP FILAMENTOVERRIDE=...) but how do I define which of my 4 materials gets this override? Is the number after the material the number of the input? In my case would it be:

;P2PP MATERIAL_DEFAULT_0_0_0
;P2PP FILAMENTOVERRIDE=PLA4
;P2PP MATERIAL_PLA_PLA_0_0_0
;P2PP MATERIAL_PLA_PLA4_0_2_0
;P2PP MATERIAL_PLA4_PLA_0_2_0 ?

Any hints, tips or examples would be greatly appreciated!

**Comment by tomvandeneede** [comment, marked as answer]
There are 2 ways
1/ with the FILAMENTOVERRIDE, you can put that in the specific filament startup code (go to the filaments settings, select the filament and enter the filament override parameter in the START-G-CODE

HOWEVER, there is a much easier way.  Under filament - Advanced there is a drop down box that contains filament types, rather than selecting one you just type the abbreviation you would like to use for the filament.   This way you can use e.g. PLA4 in the P2PP Material commands as if it has always been there



Obviously you will have to manage this manually for your special filaments


Happy printing

Tom
